pkg/kubelet/server/streaming/server.go

```diff
@@ -178,6 +177,9 @@ func (s *server) GetPortForward(req *runtimeapi.PortForwardRequest) (*runtimeapi
 	}
 	token, err := s.cache.Insert(req)
 	if err != nil {
+		if err == errCacheFull {
+			return nil, grpc.Errorf(codes.Unavailable, err.Error())
```

Comment:
what API error does this result in? ideally it would be a 429 with a retry-after